app/lib/queues/populatePracticeQueue.js

```diff
@@ -83,7 +90,26 @@ function queueSyndicationIds(q) {
     }
   });
 }
+function queueErroredIds(q) {
+  const failedIds = gpStore.getErorredIds();
+  totalRetries = failedIds.length;
+  gpStore.clearFailedIds(failedIds);
+  failedIds.forEach((id) => {
+    if (id) {
+      q.push({ id }, () => log.info(`${id} done`));
+    } else {
+      log.error('Undefined ID in syndication list');
+    }
+  });
+}
 
+function startRetryQueue(workers, drain) {
```

Comment:
Looks like there is an opportunity to refactor this function and `start` into one and pass the retry count in?